docs/faq.rst

```diff
@@ -292,3 +292,16 @@ Some examples follow:
     Please check the `link
     <https://neurostars.org/t/fmriprep-how-to-reuse-longitudinal-and-pre-run-freesurfer/4585/15>`__
     for further information on this approach.
+
+
+How to decrease *fMRIPrep* runtime when working with large datasets?
+--------------------------------------------------------------------
+*fMRIPrep* leverages PyBIDS to produce a layout, which indexes the input BIDS dataset and facilitates file queries.
+Depending on the amount of files and metadata within the BIDS dataset, this process can be time-intensive.
+As of the 20.2.0 release, *fMRIPrep* supports the ``--bids-database-dir`` option, which can be used to pass in an already indexed BIDS layout.
+
+The default *fMRIPrep* layout can be generated by running the following shell command (requires PyBIDS 0.12.1 or greater)::
+
+  pybids layout <path-to-bids-dir> <path-to-save-index> --no-validate
+
+Alternatively, a custom layout can be generated to provide finer-grain control of included and/or excluded files.
```

Comment:
I think I don't understand this. What do you mean here?

Reply:
I meant that alternative / additional `pybids layout --ignore` and `--force-index` arguments can be used to include / exclude files indexed. Given that this is a pretty rare use case, it might be best to just remove this bit

Reply:
```suggestion
By using the ``--force-index`` and ``--ignore`` options,
finer control can be achieved of what files are visible to fMRIPrep.

Note that any discrepancies between the pre-indexed database and
the BIDS dataset complicate the provenance of fMRIPrep derivatives.
If `--bids-database-dir` is used, the referenced directory should be
preserved for the sake of reporting and reproducibility.
```